lib/duckdb/library_version.rb

```diff
@@ -0,0 +1,3 @@
+module DuckDB
+  LIBRARY_VERSION = library_version[1..] if defined? library_version
```

Comment:
@Masaya-Nishijima 
Can we define `LIBRARY_VERSION` in `lib/duckdb/version.rb` file?

Reply:
I withdraw this comment. Pls ignore it.